package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "gamepad-mappings",
```

Comment:
Color me surprised that this doesn't already exist, but even still I think something along the lines of "xr-gamepad-mappings" might be more appropriate?

Reply:
makes sense.... does that mean the repo name should be xr-gamepad-mappings?

Reply:
I don't know what npm etiquette is regarding that kind of thing, but probably?